src/pages/index.html

```diff
@@ -0,0 +1,504 @@
+            <div class="col-lg-7 col-xl-8">
+                <div class="main-text">
+                    <h2>
+                        <span class="text-decoration-underline">Secure</span>
+                        your business, Secure your
+                        <span class="text-decoration-underline">future</span>
+                    </h2>
+                    <p class="text-center">
+                        Data analysis software is a type of software tool used
+                        for data analysis and reporting. It is designed to help
+                        businesses, organizations, and individuals process,
+                        visualize,
+                    </p>
+                </div>
+                <div
+                    class="d-flex justify-content-center align-items-center btn-primary-main"
+                >
+                    <button class="btn-head text-center">
+                        Get Started Free
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="dashborad">
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-10">
+                <div class="dashborad-main">
+                    <img
+                        class="w-100"
+                        src="assets/images/dashboard.png"
+                        alt=""
+                    />
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="more-than animation">
+    <div class="container">
+        <h4 class="text-center more-text">
+            More Than 10,000 users use <span>Stract</span>.
+        </h4>
+        <div class="row">
+            <div class="col-12 col-md-6 col-lg-2">
+                <div class="main-center">
+                    <img src="assets/images/Givanudan.jpg" alt="" />
+                </div>
+            </div>
+            <div class="col-12 col-md-6 col-lg-2">
+                <div class="main-center">
+                    <img src="assets/images/CERTINE.jpg" alt="" />
+                </div>
+            </div>
+            <div class="col-12 col-md-6 col-lg-2">
+                <div class="main-center">
+                    <img src="assets/images/Drome.jpg" alt="" />
+                </div>
+            </div>
+            <div class="col-12 col-md-6 col-lg-2">
+                <div class="main-center">
+                    <img src="assets/images/Google.jpg" alt="" />
+                </div>
+            </div>
+            <div class="col-12 col-md-6 col-lg-2">
+                <div class="main-center">
+                    <img src="assets/images/Amazon.jpg" alt="" />
+                </div>
+            </div>
+            <div class="col-12 col-md-6 col-lg-2">
+                <div class="main-center">
+                    <img src="assets/images/GoDaddy svg.jpg" alt="" />
+                </div>
+            </div>
+        </div>
```

Comment:
@shaban011147 @M-Khairy-B In tablet and mobile view you should add space between each logo vertically.

Reply:
Done